examples/pom.xml

```diff
@@ -35,7 +35,7 @@
     <dependency>
       <groupId>com.factset.protobuf</groupId>
   	  <artifactId>stachextensions</artifactId>
-  	  <version>1.0.0</version>
+      <version>${stach-extension-version}</version>
```

Comment:
Very minor: Please fix the indentation

Reply:
fixed https://github.com/factset/analyticsapi-engines-java-sdk/pull/65/commits/2674c712014f1fe8f9a73dbdbdcadcf635ebbe58